### ISSUE TITLE
Output dimensions

### ISSUE THREAD

**Issue by mohitsharma29** [tomrunia/PyTorchConv3D#14]
Hey

Thank you for your implementation of i3D. I was using it on my dataset with temporal durations as 16 and sample size of 112. The first change I had to make was to change Avg_Pool_5 from [8, 7, 7] to [2, 2, 4], after which I ran into another error where my output logits shape mismatched. My target size with  batch size 1 is [1,x] whereas the output returned is [1,x,1,3,1]. I don't understand how the last three dimensions are formed. Can you help me out?
